processor/transformprocessor/README.md

```diff
@@ -67,6 +67,7 @@ processors:
     traces:
       queries:
         - set(status.code, 1) where attributes["http.path"] == "/health"
+        - set(status.code, 1) where trace_state["key1"] == "val1"
```

Comment:
Instead of this example, can you add something to the "Path expressions" section above that tells users that trace_state can be used with `[]`